webpack.dev.js

```diff
@@ -221,7 +221,9 @@ module.exports = {
       },
       {
         context: ["/website"],
-        target: WEBSITE_URL,
+        bypass: (req, res, _proxyOptions) => {
```

Comment:
This is kind of unrelated, but removes an annoying 500 error in dev mode that was polluting the logs.